docs/static/scripts/Set-AvmGitHubLabels.ps1

```diff
@@ -0,0 +1,205 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingWriteHost", "", Justification = "Coloured output required in this script")]
```

Comment:
Also not suggesting a change here but I think in future we should use strict mode at version 3.0... some of the things it lets you do without it are 😨 

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/set-strictmode?view=powershell-7.3

Reply:
yeah one for later for sure